services/cinema_hall.py

```diff
@@ -0,0 +1,18 @@
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> CinemaHall:
+    return CinemaHall.objects.all()
+
+
+def create_cinema_hall(
+        hall_name: int,
+        hall_rows: int,
+        hall_seats_in_row: int
+) -> CinemaHall:
+    cinema_hall = CinemaHall.objects.create(
+        name=hall_name,
+        rows=hall_rows,
+        seats_in_row=hall_seats_in_row
+    )
+    return cinema_hall

```

Comment:
There is no need to allocate `cinema_hall` variable, return the result of the `create` operation directly. Apply this change to other similar places as well